### ISSUE TITLE
Extract train and test data form different images

### ISSUE THREAD

**Issue by sum1lim** [sum1lim/sea_ice_remote_sensing#15]
Currently, `create-dataset` script extracts data relying solely on randomness, thus train and test data can share the same source images. However, this does not allow us to test the accuracy fairly since the resulting model can be biased or overfitted on the features that indicate what image a sample came from. To solve this, test data should be extracted from the images that are not used for creating the training dataset.
